doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20-rc1.rst

```diff
@@ -0,0 +1,453 @@
+    The Fabric Synchronization enables commissioning of devices from one fabric to another without requiring manual user actions, only user consent.
+  * Dynamic SIT LIT switching support that allows the application to switch between these modes, as long as the requirements for these modes are met.
+    You can enable this using the :kconfig:option:`CONFIG_CHIP_ICD_DSLS_SUPPORT` Kconfig option.
+  * The Kconfig option :kconfig:option:`CONFIG_CHIP_ICD_SIT_SLOW_POLL_LIMIT` to limit the slow polling interval value for the device while it is in the SIT mode.
+    You can use this to limit the slow poll interval for the ICD LIT device while it is temporarily working in the SIT mode.
+  * New device types:
+
+    * Water heater
+    * Solar power
+    * Battery storage
+    * Heat pump
+    * Mounted on/off control
+    * Mounted dimmable load control
+
+* Updated:
+
+  * Thermostat cluster with support for scheduling and preset modes, like vacation, and home or away settings.
+  * Electric Vehicle Supply Equipment (EVSE) with support for user-defined charging preferences, like specifying the time when the car will be charged.
+  * Occupancy sensing cluster with features like radar, vision, and ambient sensing.
+  * Intermittently Connected Devices feature with enhancements for the Long Idle Time (LIT) and Check-In protocol.
+    With these enhancements, the state of this feature is changed from provisional to certifiable.
+
+Thread
+------
+
+* Added Kconfig options for configuring the MLE child update timeout, child supervision interval, and child supervision check timeout.
+
+Zigbee
+------
+
+* Updated:
+
+  * :ref:`nrfxlib:zboss` to v3.11.6.0 and platform v5.1.7 (``v3.11.6.0+5.1.7``).
+    They contain several fixes related to malfunctioning in a heavy traffic environment and more.
+    For details, see :ref:`zboss_changelog`.
+  * The :ref:`ZBOSS Network Co-processor Host <ug_zigbee_tools_ncp_host>` package to the new version v2.2.5.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+Machine learning
+----------------
+
+* Updated the application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+
+IPC radio firmware
+------------------
+
+* Updated the application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system in all devices that contain MRAM, such as the nRF54H Series devices.
+
+Matter bridge
+-------------
+
+* Added:
+
+  * Support for the ``UniqueID`` attribute in the Bridged Device Basic Information cluster.
+  * Version 2 of the bridged device data scheme containing ``UniqueID``.
+  * Kconfig options :ref:`CONFIG_BRIDGE_MIGRATE_PRE_2_7_0 <CONFIG_BRIDGE_MIGRATE_PRE_2_7_0>` and :ref:`CONFIG_BRIDGE_MIGRATE_VERSION_1 <CONFIG_BRIDGE_MIGRATE_VERSION_1>` to enable migration from older data schemes.
+
+nRF Desktop
```

Comment:
this part will be reviewed by @MarekPieta

Reply:
A general note, I wonder a bit if this release notes shouldn't describe changes applied since NCS 2.9?

Reply:
@Marek As this release is not supposed to be used coming from 2.9, but the path for 54h users would be 2.8 -> 2.9.0-nRF54H20-rc1, I don't think this would be ideal.